### PR TITLE
Bump to 0.9.1

### PR DESCRIPTION
**Pull request by dmp42**

### CODE REVIEW

**Comment by dmp42** [comment]
cc @shin- @stevvooe

File: docker_registry/toolkit.py [hunk 1]
```diff
@@ -299,14 +300,23 @@ def exclusive_lock(f):
     @functools.wraps(f)
     def wrapper(*args, **kwargs):
         lock_path = os.path.join(
-            '/var/lock', 'registry.{0}.lock'.format(f.func_name)
```
Comment (dmp42) [inline docker_registry/toolkit.py]: This is breaking OSX.

**Comment by stevvooe** [comment]
LGTM!

File: docker_registry/toolkit.py (hunk 1 above)
Comment (stevvooe) [inline docker_registry/toolkit.py]: Should this be a configurable path?

Comment (shin-) [inline docker_registry/toolkit.py]: People use OSX?

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ As the documentation evolves with different registry versions, be sure that befo
  * check which version of the registry you are running
  * switch to the corresponding tag to access the README that matches your product version
 
-The stable, released version is the [0.9.0 tag](https://github.com/docker/docker-registry/tree/0.9.0).
+The stable, released version is the [0.9.1 tag](https://github.com/docker/docker-registry/tree/0.9.0).
```
Comment (shin-) [inline README.md]: URL must be updated too!

File: docker_registry/images.py [hunk 3]
```diff
@@ -60,6 +62,16 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def valid_image_id(f):
```
Comment (shin-) [inline docker_registry/images.py]: Does this maybe belong in toolkit? Keep only view functions in images.py as much as possible.

**Comment by shin-** [comment]
Couple comments, otherwise LGTM

File: docker_registry/toolkit.py (hunk 1 above)
Comment (dmp42) [inline docker_registry/toolkit.py]: This is a hack to hide the fact the current SQL_ALCHEMY code is racy, so, I would rather avoid making this a first class configurable thing.

Comment (dmp42) [inline docker_registry/toolkit.py]: @shin- 🌋

File: README.md (hunk 2 above)
Comment (dmp42) [inline README.md]: Fair enough.

File: docker_registry/images.py (hunk 3 above)
Comment (dmp42) [inline docker_registry/images.py]: Again a vile hack to hide the misery... let's have this in there until further cleanup.

Comment (shin-) [inline docker_registry/images.py]: Well, this is pretty low-effort as far as improvements go, do you mind if I amend the PR with that particular change? It's not like we're getting rid of this code anytime soon.

**Comment by dmp42** [comment]
Thanks @shin- and @stevvooe!
Fixed the url version.
The rest is as-is - @shin- if you think this is acceptable, you may merge it at will :).

**Comment by dmp42** [comment]
\o/ Thanks a lot Joffrey \o/

**Comment by shin-** [comment]
LGTM!

**Pull request merged**